src/Microsoft.Data/DataFrameTable.cs

```diff
@@ -24,10 +24,10 @@ internal class DataFrameTable
 
         public DataFrameTable()
         {
-            _columns = new List<BaseDataFrameColumn>();
+            _columns = new List<BaseColumn>();
         }
 
-        private DataFrameTable(IList<BaseDataFrameColumn> columns)
+        public DataFrameTable(IList<BaseColumn> columns)
```

Comment:
Why is this now public?

Reply:
What criteria do we use to choose between public and internal visibility on members, on a public class? Assuming the member is not impelmenting an interface.

Reply:
Good question: In this case it doesn't matter if this constructor public or internal. I just chose public, there was no particular reason to do so.